### ISSUE TITLE
Error icon when adding applet to panel

### ISSUE THREAD

**Issue by fredcw** [linuxmint/cinnamon#10054]
* Cinnamon version 4.8.6
  
 * Distribution - Mint 20.1 
 
**Issue**

![Screenshot](https://user-images.githubusercontent.com/58893963/115150987-ceb2de00-a062-11eb-9ed9-75ece1ed386e.png)

Error Icon appears when adding applet back to panel even though applet works as normal.

**Steps to reproduce**

Add any applet to panel, remove it, then add it back. Error icon appears.

**Comment by mtwebster** [comment]
Is there anything in .xsession-errors?

**Comment by fredcw** [comment]
No, just the usual. The .xsession meassages are the same whether the icon appears or not.

```
js-Message: 16:30:53.103: JS LOG: [LookingGlass/info] Reloading applet: notifications@cinnamon.org/7
Cjs-Message: 16:30:53.104: JS LOG: [LookingGlass/info] Reloading applet: systray@cinnamon.org/11
Cjs-Message: 16:30:53.105: JS LOG: [LookingGlass/info] Reloading applet: removable-drives@cinnamon.org/13
Cjs-Message: 16:30:53.106: JS LOG: [LookingGlass/info] Reloading applet: network@cinnamon.org/14
Cjs-Message: 16:30:53.107: JS LOG: [LookingGlass/info] Reloading applet: sound@cinnamon.org/15
Cjs-Message: 16:30:53.129: JS LOG: [LookingGlass/info] Reloading applet: inhibit@cinnamon.org/17
Cjs-Message: 16:30:53.130: JS LOG: [LookingGlass/info] Reloading applet: calendar@cinnamon.org/18
Cjs-Message: 16:30:53.130: JS LOG: [LookingGlass/info] Reloading applet: power@cinnamon.org/53
Cjs-Message: 16:30:53.132: JS LOG: [LookingGlass/info] Reloading applet: radio@driglu4it/326
Cjs-Message: 16:30:53.133: JS LOG: [LookingGlass/info] Reloading applet: CinnVIIStarkMenu@NikoKrause/334
Cjs-Message: 16:30:53.153: JS LOG: [LookingGlass/info] Adding systray: network (24x24px)
Cjs-Message: 16:30:53.153: JS LOG: [LookingGlass/info] Resized network with normalized size (32x32px)
Cjs-Message: 16:30:53.164: JS LOG: [LookingGlass/info] Hiding systray: network
Cjs-Message: 16:30:53.164: JS LOG: [LookingGlass/info] Hiding systray: network
Cjs-Message: 16:30:53.221: JS LOG: [LookingGlass/info] Hiding systray: network
Cjs-Message: 16:30:53.221: JS LOG: [LookingGlass/info] Hiding systray: network
Cjs-Message: 16:30:53.571: JS LOG: Unknown network device type, is 14

** (csd-power:722): CRITICAL **: 16:30:53.572: abs_to_percentage: assertion 'max > min' failed

```

**Comment by fredcw** [comment]
Nothing when run from command line:

```
Using pam module (python3-pampy)
Loading Applets module
```

**Comment by mtwebster** [comment]
Are you using the current cinnamon release? Or daily builds/git?

If you look at the settings key that stores enabled applets, are there any there aren't actually installed (or expected to be enabled)?
```
gsettings get org.cinnamon enabled-applets
```

There's a PR that may have an impact on this as well:
https://github.com/linuxmint/cinnamon/pull/9940

**Comment by fredcw** [comment]
Yes, I tested it on mint 20.1 with daily builds PPA. It also happens on Manjaro.

>are there any there aren't actually installed

No, all applets in the list are on the panel.

Not sure if this is related to 9940, this happen when *any* applet is added to the panel.

**Comment by fredcw** [comment]
This issue appears to have been fixed by #10071
